### ISSUE TITLE
Add portable version

### ISSUE THREAD

**Issue by jhekasoft** [jhekasoft/insteadman3#8]

**Comment by eaglexk** [comment]
Да, реально не хватает портативной версии. Выглядит шикарно, удобнее instead-launcher, но заменить его не могу потому, что проект не портативный. Хотя кажется, чего же стоит добавить позможность указать папку с играми и папку хранения когфига?
